studies/models.py

```diff
@@ -281,6 +281,10 @@ def default_study_structure():
     return {"frames": {}, "sequence": []}
 
 
+def default_exit_url():
+    return f"{settings.BASE_URL}/studies/history/"
+
```

Comment:
Instead of a static URL, could we use "reverse" to generate the path and concatenate it with the base?

Reply:
Yes! Great suggestion, thank you!